web/packages/extension/src/background.ts

```diff
@@ -2,15 +2,21 @@ import * as utils from "./utils";
 import { isMessage } from "./messages";
 
 async function contentScriptRegistered() {
-    const matchingScripts = await chrome.scripting.getRegisteredContentScripts({
+    const matchingScripts = await utils.scripting.getRegisteredContentScripts({
         ids: ["plugin-polyfill"],
     });
-    return matchingScripts.length > 0;
+    return matchingScripts?.length > 0;
 }
 
 async function enable() {
-    if (chrome?.scripting && !(await contentScriptRegistered())) {
-        await chrome.scripting.registerContentScripts([
+    if (
+        utils.scripting.ExecutionWorld &&
```

Comment:
Just to document some things that wouldn't quite work right on a version of Firefox that old:

https://github.com/ruffle-rs/ruffle/blob/2b17b6ce20591b958b879e27182ad5d0c5712084/web/packages/core/src/shadow-template.ts#L326
- https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content#browser_compatibility without the `-moz-` prefix this was not supported until Firefox 94, and we don't have a `-moz-` fallback. All this causes is the modals for the save manager and volume controls to be the full width of the screen, so not a big deal.

https://github.com/ruffle-rs/ruffle/blob/2b17b6ce20591b958b879e27182ad5d0c5712084/web/packages/core/src/shadow-template.ts#L293
- https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio#browser_compatibility was not supported until Firefox 89. Again, just a minor CSS styling issue, so not a big deal.

That said, aside from these minor CSS issues, I do think that version of Firefox would work, but I haven't tested it.

This could probably just be `utils.scripting?.ExecutionWorld`.